src/java.desktop/share/classes/javax/swing/filechooser/FileSystemView.java

```diff
@@ -264,15 +264,15 @@ public Icon getSystemIcon(File f) {
     * {@code ShellFolder} class. Whenever possible, the icon
     * returned will be a multi-resolution icon image,
     * which will allow better scaling with different
-    * magnification factors.
+    * scaling factors.
```

Comment:
“…which will allow better support for High DPI environments with different scaling factors.” — does it look better?
“scaling with … scaling factors” doesn't sound good to me.

Reply:
Yes, much better. I updated the wording and once the PR is approved i will fix it in CSR too.